demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -934,6 +934,33 @@ def is_valid_parameters_display_name(self) -> bool:
 
         return True
 
+    def is_valid_integration_file_path(self) -> bool:
+        absolute_file_path = self.file_path
+        integrations_folder = os.path.basename(os.path.dirname(absolute_file_path))
+        integration_file = os.path.basename(absolute_file_path)
+
+        # drop file extension
+        integration_file, _ = os.path.splitext(integration_file)
+
+        if integrations_folder == 'Integrations':
+            if not integration_file.startswith('integration-'):
+
+                error_message, error_code = \
+                    Errors.is_valid_integration_file_path_in_integrations_folder(integration_file)
+                if self.handle_error(error_message, error_code, file_path=self.file_path):
+                    return False
+            return True
```

Comment:
drop this return and change the if in 954 to elif

Reply:
done